src/dynamixel_main.cpp

```diff
@@ -0,0 +1,123 @@
+#include <fcntl.h>
+#include <termios.h>
+#define STDIN_FILENO 0
+
+#include <stdio.h>
+#include <chrono>
+#include <functional>
+#include <memory>
+#include <string>
+
+#include "dynamixel_sdk/dynamixel_sdk.h"
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+
+#define PROTOCOL_VERSION                2.0
```

Comment:
Don't need to define this, we all know that the OpenCR will only work with protocol version 2.0.